extensions/cornerstone/src/index.js

```diff
@@ -13,6 +18,9 @@ export default {
    */
   id: 'cornerstone',
 
+  preRegistration(configuration = {}) {
```

Comment:
It might just be worth doccing that either an object can be passed here (global config), or an array of config for each `cornerstoneTools` module?